pkgs/by-name/gr/gruvbox-plus-icons/package.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation {
+  pname = "gruvbox-plus-icons";
+  version = "2023-11-30";
+
+  src = fetchFromGitHub {
+    owner = "SylEleuth";
+    repo = "gruvbox-plus-icon-pack";
+    rev = "5ce3ef1ae9d8360e4aadfcf73842df9a417dd53b";
+    hash = "sha256-xS6ijyRhc9CaZVERLjqebbNsSbPoFUVTAutOjWiOUKc=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/share/icons
+    cp -r Gruvbox-Plus-Dark $out/share/icons/
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Gruvbox Plus icon pack for Linux desktops based on the Gruvbox color theme";
```

```suggestion
    description = "Icon pack for Linux desktops based on the Gruvbox color theme";
```

I missed this earlier, desc shouldn't have the package name